### ISSUE TITLE
Plasma not burning properly

### ISSUE THREAD

**Issue by Artemiuz28** [tgstation/tgstation#32337]
Seemingly, igniter in Toxins mixing chamber on Pubby is not powerful enough to ignite the plasma/oxygen mix. I've tried both 66/33 and 50/50. Only a single-tile fire appears, but doesn't spread.

**Comment by duncathan** [comment]
@as334 a few people seem to be commenting on fire being weaker than expected, thoughts?

**Comment by Artemiuz28** [comment]
Uhmm, I played before the new gases being added, and 66/33 mix in the chamber ignited easily. Now, it doesn't

**Comment by Artemiuz28** [comment]
Also, tossing in a few igniter-remote assemblies and signaling them still doesn't help. The fire gets put out.

**Comment by Artemiuz28** [comment]
I just ran a server on BoxStation, and the problem remains. So, it's not about the igniter, but about the plasma. Please fix

**Comment by duncathan** [comment]
Thank you for the continued testing. There was recently a major update to atmos reactions including fire, but I don't believe that plasma fires becoming significantly weaker like this is intended.

**Comment by AnturK** [comment]
```
runtime error: 
[07:42:10]cannot read from list
[07:42:10]proc name: process (/obj/effect/hotspot/process)
[07:42:10]  source file: LINDA_fire.dm,115
[07:42:10]  usr: null
[07:42:10]  src: the hotspot (/obj/effect/hotspot)
[07:42:10]  src.loc: the floor (89,141,2) (/turf/open/floor/plasteel)
[07:42:10]  call stack:
[07:42:10]the hotspot (/obj/effect/hotspot): process()
[07:42:10]Atmospherics (/datum/controller/subsystem/air): process hotspots(0)
[07:42:10]Atmospherics (/datum/controller/subsystem/air): fire(0)
[07:42:10]Atmospherics (/datum/controller/subsystem/air): ignite(1)
[07:42:10]Master (/datum/controller/master): RunQueue()
[07:42:10]Master (/datum/controller/master): Loop()
[07:42:10]Master (/datum/controller/master): StartProcessing(0)
```

Hotspots seem to runtime due to no gas asserts, but dunno if that's the cause.

**Comment by Morgan-Edlund-1516** [comment]
Pretty sure this was caused by the nitro name change PR, since that broke stimulum's gas reaction as well

**Comment by Robustin** [comment]
Yea I'm seeing this issue, brand new since yesterday, 50/50 plasma oxygen mixes heated above 200c aren't burning internally anymore, rooms filled with ideal burn mixes wont burn more than a tile beyond a hotspot. Somethings fucked in the last 24hrs.

**Comment by Jalleo** [comment]
@duncathan This could be the cause for it but might not be. Merged 7 days ago now https://github.com/tgstation/tgstation/pull/30791/files#diff-daf90c5e6bc314304688cbd4059754b2L181

**Comment by Gouty** [comment]
I set up the most perfect plasma flood, it was beautiful, I was an AI that was given laws to burn the station down, plasma/oxygen mix everywhere BUT IT WOULD NOT BURN!!

not even the holodeck atmos burn test would catch alight.

**Comment by AnturK** [comment]
Is this a thing after #32353 ?

**Comment by Cyberboss** [comment]
I don't think so @Jalleo, that seems like it'd have the opposite effect

**Comment by davethwave** [comment]
perhaps the co2 or n2 or whatever the fire naturally makes kills the fire too fast for it to spread for some reason?

**Comment by Supermichael777** [comment]
He deleted the fire defines. For some reason

**Comment by ninjanomnom** [comment]
This should be fixed and can't seem to be reproduced after #32353
